tests/ci/cpu_unit_tests.yml

```diff
@@ -0,0 +1,101 @@
+# cpu_unit_tests.yml
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+#
+#
+# NOTE: start with cpu to make sure everything still works
+#
+# use variable group name
+variables:
+- group: AzureKeyVaultSecrets
+- name  : 'test'
+  value : 'tests/ci/run_pytest.py'
+- name  : 'testfolder'
+  value : './tests/unit'
+- name  : 'testmarkers'
+  value : '"not notebooks and not spark and not gpu"'
+- name  : 'junitxml'
+  value : '"reports/test-unit.xml"'
+#   
+- name  : 'maxnodes'
+  value : 4
+#- name  : 'rg'
+#  value : 'recommender'
+#- name  : 'wsname'
+#  value : 'RecoWS'
+- name  : 'reponame'
+  value : 'Recommenders'
+- name  : 'branch'
+  value : 'azure-pipelines-bz'
+- name  : 'clustername'
+  value : 'Reco-cpu-test'
+# GPU
+#- name  : 'vmsize'
+#  value : 'STANDARD_NC6'
+# CPU
+- name  : 'vmsize'
+  value : 'STANDARD_D3_V2'
+- name  : 'dockerproc'
+  value : 'cpu'
+- name  : 'expname'
+  value : 'cpu_unit_tests'
+
+#trigger:

```

Comment:
do we need to keep this? should it change to the correct branch?

Reply:
good point. I changed it to trigger none so it will trigger off PRs and not branch changes too. Although, we may want to talk about adding a dev branch trigger for use by Microsoft devs at some point.

Reply:
done.